public/dashboard.css

```diff
@@ -38,123 +48,149 @@ div.stat-box {
 }
 
 span.stat {
-  font-size: 5vh;
+  font-family: 'Roboto Mono', monospace;
```

Comment:
why monospace for these? with numbers in the tens and hundreds I think it's gonna get squishy, and the spaces around the slashes end up taking up too much space.